spec/features/gym_search/index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'gyms near me page' do
+  before do
+    visit dashboard_path(@user.id)
+
+    click_on 'Find Gyms Near Me'
+  end
+
+  it 'can find gyms near me', :vcr do
+    expect(page).to have_content("Gyms Near #{@user.zip_code}")
+
+    within '#gyms' do
+      expect(page).to have_css('div', count: 20)
```

Comment:
Nice check on how many results are returned!  Probably a stretch goal/feature, but pagination on the results could be cool.